### DISCUSSION TITLE
Router or switch?   Reason 10

### DISCUSSION THREAD

**Discussion by RobinSchell** [apohl79/audiogridder#342] [Q&A]
I have two basic questions: 
1. Is AG working well with propellerhead Reason 10?  (99.9 % of VSTs work fine but some, like Airwindows for example, have issues due to specifics about how Reason implements VSTs)

2. Best way to network my two machines:  I have a PC win win 7 running Reason 10, and just got a refurbed win 10 i5 machine. My main computer is currently connected via a (very long) LAN cable to a router in order have broadband access - the router is way too far away to simply run another cable to the other machine. So should I get a small switch or another router so that both machines are networked together and both have broadband access?  (keeping in mind that I may want to add another server machine)
Please forgive my ignorance regarding networking, it is not something I have previously had to do :)

**Comment by CaBuffalo** [comment]
1, I have no experience with Reason. You might have to try it and find out.
2, yes, put a switch between the long cable you have and the two systems.
That'll give you both pc broadband access and ability to communicate on
LAN. AG will work well on LAN.

On Sat, Jan 30, 2021 at 12:18 PM RobinSchell <notifications@github.com>
wrote:

> I have two basic questions:
>
>    1.
>
>    Is AG working well with propellerhead Reason 10? (99.9 % of VSTs work
>    fine but some, like Airwindows for example, have issues due to specifics
>    about how Reason implements VSTs)
>    2.
>
>    Best way to network my two machines: I have a PC win win 7 running
>    Reason 10, and just got a refurbed win 10 i5 machine. My main computer is
>    currently connected via a (very long) LAN cable to a router in order have
>    broadband access - the router is way too far away to simply run another
>    cable to the other machine. So should I get a small switch or another
>    router so that both machines are networked together and both have broadband
>    access? (keeping in mind that I may want to add another server machine)
>    Please forgive my ignorance regarding networking, it is not something
>    I have previously had to do :)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/342>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BHRC57HXQ43MCFBADS4RSQ5ANCNFSM4W2T6NLA>
> .
>


-- 
Kevin Ma
Supported by Gmail.
